packages/o-spreadsheet-engine/src/plugins/ui_core_views/header_sizes_ui.ts

```diff
@@ -111,8 +111,11 @@ export class HeaderSizeUIPlugin extends CoreViewPlugin<HeaderSizeState> implemen
         break;
       case "SET_FORMATTING":
         if (
-          cmd.style &&
-          ("fontSize" in cmd.style || "wrapping" in cmd.style || "rotation" in cmd.style)
+          "style" in cmd &&
```

Comment:
careful with that. Because in a collaborative context, `style: undefined` is completely removed from the command when it's serialized to be sent over the network